docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -93,6 +93,8 @@ yourself. The version must include the `fw_cfg_name` feature as well as the expe
 
 When building from source, you have to do a final `mv $GOPATH/bin/terraform-provider-libvirt ~/.terraform.d/plugins/terraform-provider-libvirt_v0.5.3`:
 
+When building from source, you will need to have `libvirt-dev` package installed.
```

Comment:
Thanks, I remember there was some sentence like this in the original PR draft but then we switched to the release version and removed it.
Can you tell which distro the package name refers to? E.g., I would say `libvirt-dev` for Debian/Ubuntu and `libvirt-devel` for Fedora and similar.

Reply:
@pothos updated the PR with correct package for Debian/Ubuntu and Fedora/Centos distros